terraform/modules/admin-lb/main.tf

```diff
@@ -30,13 +30,37 @@ resource "kubernetes_manifest" "admin-server-cert" {
   }
 }
 
+resource "null_resource" "wait_for_server_cert_creation" {
+  provisioner "local-exec" {
+    command = <<-EOT
+    COUNTER=0
+    MAX_TRIES=50
+    while ! kubectl describe managedcertificate ${var.admin_server_cert_id} --namespace=${var.deployment_namespace} | grep -i "certificate name" && [ $COUNTER -lt $MAX_TRIES ]
+    do
+      sleep 15
+      COUNTER=$((COUNTER + 1))
+    done
+    if [ $COUNTER -eq $MAX_TRIES ]; then
+      echo "The managed server certificate was not created, terraform can not continue!"
```

Comment:
What to do in this case? Can you extend this error message, if possible?

Reply:
add further instruction on what to do.